src/components/TagInput/TagInput.js

```diff
@@ -133,19 +136,24 @@ export default class TagInput extends Component {
         : [{ name: `no more than ${maxTags} allowed`, isError: true }]
       : suggestions
 
-    return <div className={cx(theme.root, { [theme.readOnly]: readOnly }, className)} onClick={this.focus}>
-      <ul className={theme.selected}>
+    return <div className={cx(theme.root, { [theme.readOnly]: readOnly }, className)} styleName='list-outer-container' onClick={this.focus}>
+      <ul className={theme.selected} styleName='list-selected'>
         {selectedItems}
       </ul>
-      <div className={theme.search}>
-        <div className={theme.searchInput}>
+      <div className={theme.search} styleName='list-input-container'>
+        <div className={theme.searchInput} styleName='list-inner-container'>
           <input
             className={theme.searchInput}
-            styleName={cx({ 'error': maxReached })}
+            styleName={cx({ 'error': maxReached }, 'list-input')}
             ref={this.input}
             type='text'
             placeholder={placeholder}
             spellCheck={false}
+            onClick={() => { this.handleChange('') }}
```

Comment:
should this be onFocus? What they if tab into the field? actually im curious what this is needed for too

Reply:
yup onFocus is better, I just wasn't thinking about tabbing